tests/arm/Tests.lst

```diff
@@ -10017,7 +10017,7 @@ RelativePath=CoreMangLib\cti\system\threading\interlocked\InterlockedIncrement1\
 WorkingDir=CoreMangLib\cti\system\threading\interlocked\InterlockedIncrement1
 Expected=0
 MaxAllowedDurationSeconds=600
-Categories=EXPECTED_PASS
+Categories=EXPECTED_PASS;JITSTRESS_FAIL
```

Comment:
why `JITSTRESS_FAIL` and not `GCSTRESS_FAIL`? Should this be a different change?